### ISSUE TITLE
bug(MultiSelect): does not open menu on enter

### ISSUE THREAD

**Issue by abbeyhrt** [carbon-design-system/carbon#3861]
**Description**
Tab to the Multiselect box
Hit the Enter

The Enter key does not expand or collapse the listbox.

See ARIA Authoring Practices Guide for additional information. 

It states the following:

Enter key:

If the focus is on the button, expands the listbox and places focus on the currently selected option in the list.
If focus is in the listbox , collapses the listbox and keeps the currently selected option as the button label.

-->

## Environment

> Operating system
macOs Mojave Version 10.14.6
> Browser
Chrome

## Detailed description

> What version of the Carbon Design System are you using?
v10

> What WCAG 2.1 checkpoint does the issue violate?
I don't think this technically violates WCAG 2.1. A user can still access the menu with their keyboard, just with the spacebar and not the enter key.

**Comment by asudoh** [comment]
@abbeyhrt Can this be closed or do you have more work to do? Thanks!

**Comment by abbeyhrt** [comment]
Not yet! Still have some work to do on it

**Comment by asudoh** [comment]
Closing given this is not a hard requirement based on our discussion.
